scripts/docgen/content-sources/js/toc.yaml

```diff
@@ -67,31 +67,31 @@ toc:
   - title: "IdTokenResult"
     path: /docs/reference/js/firebase.auth.IDTokenResult
   - title: "MultiFactorAssertion"
-    path: /docs/reference/js/firebase.auth.multifactorassertion
+    path: /docs/reference/js/firebase.auth.MultiFactorAssertion
```

Comment:
I think these changes might actually break the links, won't they?

The current lowercase is working; but when I manually change a URL to camel case, I get 404s.

Reply:
This will change the case of the generated filename, when the docs are published, they will work. This is the result of running docgen locally after this change:

<img width="342" alt="Screen Shot 2020-12-01 at 12 05 08 PM" src="https://user-images.githubusercontent.com/10716545/100791079-98373800-33cd-11eb-9ff2-0378967727f1.png">

Reply:
Got it, thanks!